index.html

```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
             </div>
             
             <div class="first-card">
-                <img class="first-project" src="img/top-proj.png" alt="project-screenshot">
+                <div class="first-project"></div>
```

Comment:
- [ ] I think this issue is still not fixed yet. See the screenshot of your project and please try to increase the width of the cards for the desktop version.

Your Project design|
:------------------
![a1](https://user-images.githubusercontent.com/39523312/160237121-4273fda2-e215-4de8-bccc-94dcc841943e.png)
-------:|

Reply:
@mmsesay I fixed it. kindly check